dist/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Weather App</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="card">
+        <div class="search">
+            <input type="text" 
+            placeholder="Enter the name of City"
+            >
+            <button class="btn">Submit</button>
+        </div>
+        <div class="weather">
+            <img src="rain.png" class="Weather Icon">
```

Comment:
`img` tag should always contain `alt` attribute

Reply:
Fixed